### PR TITLE
Add csv download on legend component

### PR DESCRIPTION
**Pull request by kahummer**
The pr adds a download button on the legend to enable data download in csv format
Screenshot from 2020-06-12 17-34-46

### CODE REVIEW

**Comment by cKellyDesign** [comment]
@kahummer does this PR have a ticket? If so please link, if not please add description above.

**Comment by cKellyDesign** [comment]
@kahummer what's the status of this PR?

**Comment by kahummer** [comment]
@cKellyDesign I think once I fix the conflicts it can be merged

**Comment by cKellyDesign** [comment]
> Once I fix the conflicts it can be merged

@kahummer please request reviews when the conflicts are resolved 🙏

File: src/lib/components/Legend/Legend.js [hunk 1]
```diff
@@ -210,6 +213,7 @@ export class Legend extends React.Component {
     latestTimestamp = latestTimestamp ? <span>Timestamp: {latestTimestamp}</span> : null;
     let primaryLegend;
     let layer;
+    let exportLink = (<a className="glyphicon glyphicon-download-alt" data-download="download" target="_blank" href={layerObj.exportLink} download></a>);
```
Comment (ukanga) [inline src/lib/components/Legend/Legend.js:216]: Should the export link be a component on its own? Or is it overkill in this case?

Comment (kahummer) [reply src/lib/components/Legend/Legend.js:216]: I thought it would be an overkill since it's a single anchor tag

**Review by kelvin-muchiri — APPROVED** [review]

**Pull request merged**